CONTRIBUTING.md

```diff
@@ -1,7 +1,57 @@
+# For Contributors
+
 GraalVM welcomes contributors to the core platform and projects that extend that
 platform. There have been significant contributions from both industry
 and academia so far and we thank you for considering to contribute your changes!
 
+### How to Contribute
+
 - Learn [how to become a GraalVM contributor](https://www.graalvm.org/community/contributors/).
   - Check individual README.md and CONTRIBUTING.md files in the subprojects to learn how to build and import them into your IDE (for example, [the compiler README.md](compiler/README.md))
 - Subscribe and post to [graalvm-dev@oss.oracle.com](https://oss.oracle.com/mailman/listinfo/graalvm-dev) for questions related to working with the sources or extending the GraalVM ecosystem by creating new languages, tools, or embeddings.
+
+### Contributor Roles
+
+There are different roles for contributors of the project. Find a list of current contributors in [CENSUS.md](CENSUS.md).
+
+* Committer
+  * Has Signed the [Oracle Contributor Agreement](https://oca.opensource.oracle.com/), which is a prerequisite for contributing to the project
+  * At least one pull request authored by the individual has been merged
+
+* Code Owner
+  * Source code directories contain an OWNERS.toml metadata file to define one or more Code Owners.
```

Comment:
FWIW Github supports [`CODEOWNERS` files](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) which can be used to enforce the review requirement on github.

Reply:
That is a good suggestion, I will check if we can reuse this mechanism.